zerver/webhooks/yo/doc.md

```diff
@@ -0,0 +1,28 @@
+See your Yo App notifications in Zulip!
+
+First, on your {{ settings_html|safe }}, create a bot for
+{{ integration_display_name }}. Construct the URL for the
+{{ integration_display_name }} bot using the bot API key
+and the email address associated with your Zulip account:
+
```

Comment:
Same thing, I'd use `create-stream` here and move the mention about the "email address" to a separate line. :)